package.json

```diff
@@ -19,8 +19,8 @@
 		"dev:all": "concurrently --names=api,ui \"pnpm -w dev:api\" \"pnpm -w dev:ui\"",
 		"dev:api": "nx run @pcgl-daco/api:dev",
 		"dev:ui": "nx run @pcgl-daco/ui:dev",
-		"setup:all": "nx run-many --all --target=init",
-		"test": "nx run-many --all --target=test",
+		"setup:all": "nx run-many --all --target=setup",
+		"test:all": "nx run-many --all --target=test",
```

Comment:
There's additional relevant changes in api package.json: `"setup": "pnpm init:env && pnpm init:db",`

Reply:
Thanks, updating. I had originally named the scripts init but it conficts with the default npm init comand so I changed them last minute... inconsistently apparently.